requirements_pt.txt

```diff
@@ -1,8 +1,8 @@
 numba
-numpy < 2.0
-peft
-prettytable
-psutil
-py-cpuinfo
-pydantic
+numpy==1.23.5 ; python_version < '3.12'
+numpy<2.0 ; python_version >= '3.12'
```

Comment:
nit: there are two version limitations for the `numpy`.

Reply:
from Habana team.